scripts/run_backend_tests_test.py

```diff
@@ -417,11 +443,37 @@ def test_incomplete_coverage_is_displayed_correctly(self) -> None:
         task_to_taskspec[tasks[0]] = run_backend_tests.TestingTaskSpec(
             test_target, True)
 
+        with self.print_swap:
+            run_backend_tests.print_coverage_report(
+                tasks, task_to_taskspec)
+        self.assertIn(
+            'INCOMPLETE PER-FILE COVERAGE (98%%): %s' %
+            test_target, self.print_arr)
+
+    def test_cancelled_tests_coverage_is_displayed_correctly(self) -> None:
```

Comment:
*Ditto

Reply:
It tests if cancelled tests will be printed.

Reply:
seems this test is not neccessary, I will delete this test.